### ISSUE TITLE
[Briefcase] Add documentation about the new GeoJSON companion export files

### ISSUE THREAD

**Issue by ggalmazor** [opendatakit/docs#894]
Once https://github.com/opendatakit/briefcase/pull/669 is merged, we should add information about the new export type option and the output format, including the new CLI flag

**Comment by ggalmazor** [comment]
Closing in favor of #913
